internal/runners/mcp/rebuildproject/rebuildproject.go

```diff
@@ -0,0 +1,88 @@
+package rebuildproject
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runbits/commits_runbit"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	bpModel "github.com/ActiveState/cli/pkg/platform/model/buildplanner"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+type ProjectErrorsRunner struct {
+	auth     *authentication.Auth
+	output   output.Outputer
+	svcModel *model.SvcModel
+}
+
+func New(p *primer.Values) *ProjectErrorsRunner {
+	return &ProjectErrorsRunner{
+		auth:     p.Auth(),
+		output:   p.Output(),
+		svcModel: p.SvcModel(),
+	}
+}
+
+type Params struct {
+	project *project.Namespaced
+}
+
+func NewParams(project *project.Namespaced) *Params {
+	return &Params{
+		project: project,
+	}
+}
+
+func (runner *ProjectErrorsRunner) Run(params *Params) error {
+	branch, err := model.DefaultBranchForProjectName(params.project.Owner, params.project.Project)
+	if err != nil {
+		return fmt.Errorf("error fetching default branch: %w", err)
+	}
+
+	// Collect "before" buildplan
+	bpm := bpModel.NewBuildPlannerModel(runner.auth, runner.svcModel)
+	localCommit, err := bpm.FetchCommitNoPoll(*branch.CommitID, params.project.Owner, params.project.Project, nil)
+	if err != nil {
+		return errs.Wrap(err, "Failed to fetch build result")
+	}
+
+	// Collect "after" buildplan
+	bumpedBS, err := localCommit.BuildScript().Clone()
+	if err != nil {
+		return errs.Wrap(err, "Failed to clone build script")
+	}
+
+	now := captain.TimeValue{}
+	now.Set("now")
+	ts, err := commits_runbit.ExpandTime(&now, runner.auth)
+	if err != nil {
+		return errs.Wrap(err, "Failed to fetch latest timestamp")
+	}
+	bumpedBS.SetAtTime(ts, true)
```

Comment:
This is for facilitating flag parsing, it's a roundabout way of grabbing the time if you already know that you want "now". In this case you can just call the underlying logic:

https://github.com/ActiveState/cli/blob/7301bd99e67934be0c41312b503446e2e9025a69/internal/runbits/commits_runbit/time.go#L32-L37

I wouldn't expect to see any use of the captain package in mcp tooling, that's specifically for command line handling, so a red flag.